package.json

```diff
@@ -59,23 +59,23 @@
     "TypeScript"
   ],
   "dependencies": {
-    "@typescript-eslint/parser": "^4.0.0",
-    "eslint-config-standard": "^16.0.0"
+    "@typescript-eslint/parser": "^4.15.2",
+    "eslint-config-standard": "^16.0.2"
   },
   "peerDependencies": {
     "eslint": ">=7.12.1",
     "eslint-plugin-import": ">=2.22.1",
     "eslint-plugin-node": ">=11.1.0",
     "eslint-plugin-promise": ">=4.2.1",
     "typescript": ">=3.9",
-    "@typescript-eslint/eslint-plugin": ">=4.0.1"
+    "@typescript-eslint/eslint-plugin": ">=4.15.2"
```

Comment:
We should not bump this without a good reason. It's a breaking change. A good reason would be using a new rule.

Reply:
Indeed, do you know how we could update `@typescript-eslint/eslint-plugin` in `devDependencies` without changing the version in `peerDependencies`, because it seems like if we do it breaks the CI.

Reply:
The test that would be the most useful is one that proves that the peer dep of the plugin starts with `>=` and is the minimal version that includes all of the rules that we're using, and all of the specific rule configuration options and values that we're setting.